projects/admin-core/src/lib/catalog/feature-type-selector/feature-type-selector.component.ts

```diff
@@ -58,6 +59,9 @@ export class FeatureTypeSelectorComponent implements OnInit, OnDestroy {
     }
   }
 
+  @Input()
+  public excludedFeatureSourceProtocols: Array<string> | null | undefined;
```

Comment:
Looks good, one minor thing, here you define the input with Array of strings. Since protocol is a typed thing it would be slightly better to use een Array of FeatureSourceProtocolEnum `FeatureSourceProtocolEnum[]` instead.